k256/src/ecdsa/recoverable.rs

```diff
@@ -362,7 +365,7 @@ mod tests {
     fn public_key_recovery() {
         for vector in VECTORS {
             let sig = Signature::try_from(&vector.sig[..]).unwrap();
-            let prehash = Sha256::new().chain(vector.msg);
+            let prehash = Sha256::new().chain_update(vector.msg);
```

Comment:
`new_with_update` should work here too

Reply:
Done (you meant `new_with_prefix` I assumed).